api/server.spec.js

```diff
@@ -1,6 +1,8 @@
 const mongoose = require('mongoose');
 
 const Game = require('./games/Game');
```

Comment:
The path should be `'../games/Game'` since the games directory is located at the same level as the api directory.